apps/src/studio/blocks.js

```diff
@@ -3208,9 +3208,9 @@ exports.install = function (blockly, blockInstallOptions) {
       this.id +
       "', " +
       (this.getFieldValue('SPRITE') || '0') +
-      ", '" +
-      (this.getFieldValue('VALUE') || ' ') +
-      "');\n"
+      ', ' +
+      JSON.stringify(this.getFieldValue('VALUE') || ' ') +
+      ');\n'
```

Comment:
I see some of `studio_saySpriteChoices` still uses double quotes (`"`), should we cleanup the rest of the method?
```
"Studio.saySprite('block_id_" +
      this.id +
      "', " +
      (this.getFieldValue('SPRITE') || '0') +
```

Reply:
I believe it's fine if I understand you correctly.

<img width="700" alt="Screenshot 2024-02-29 at 20 27 45" src="https://github.com/code-dot-org/code-dot-org/assets/11708250/ea1d442a-45fd-4289-ba27-de4b38508735">